src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1516,13 +1507,13 @@ Take a moment to preview the various utilities listed or view our comprehensive
     <value>Snap a window to multiple zones by holding the **Ctrl** key (while also holding **Shift**) when dragging a window.</value>
   </data>
   <data name="Oobe_FileLocksmith_HowToUse.Text" xml:space="preserve">
-    <value>In File Explorer, **right-clicking one or more selected files** and **clicking on "What's using this file?"** from the context menu.</value>
+    <value>In File Explorer, right-click one or more selected files and select **What's using this file?** from the context menu.</value>
```

Comment:
Side note: the "What's using this file?" context menu entry isn't great - and feels weird compared to other context menu items that refer to actions.

E.g. "See blocking processes" or something like that would be more appropriate.

Reply:
My preference would be something like "\<action> with \<program>". So, let's say "Investigate with File Locksmith"?
Or we could work up to putting all "our" Explorer context menu items in one sub-folder. Should be easy on Win 11 (= #21719, related to #13168, #22028) but also possible on Win 10.

```
(...)
PowerToys   >
    Image Resizer
    File LockSmith
    Power Rename
(...)
```